src/prx/helpers.py

```diff
@@ -168,6 +179,19 @@ def repair_with_gfzrnx(file):
     )
     if result.returncode == 0:
         log.info(f"Ran gfzrnx file repair on {file}")
+        with open(file, "r") as f:
```

Comment:
otherwise, running gfzrnx repair creates a file with a different content hash each time, and we never have cache hits.